easybuild/easyconfigs/l/libxml2/libxml2-2.9.3-foss-2015a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxml2'
+version = '2.9.3'
+
+homepage = 'http://xmlsoft.org/'
+description = """Libxml2 is the XML C parser and toolchain developed for the Gnome project (but usable
+ outside of the Gnome platform)."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'http://xmlsoft.org/sources/',
+    'http://xmlsoft.org/sources/old/'
+]
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = 'CC="$CC" CXX="$CXX" --with-pic --without-python'
```

Comment:
@valtandor: please include `--with-zlib=$EBROOTZLIB` in here at the end, see #2551 and #2555